data/tweaks/cleanup-old-kernels/metadata.yml

```diff
@@ -0,0 +1,8 @@
+description: "Uninstall Old kernels"
+type: script
+stability: stable
+
+warning: |-
```

Comment:
is there a distinction between warnings and notes per se?

Reply:
warnings are custom messages to show in the warning prompt, the description is meant to describe the thing, there is no title.